gradlew

```diff
@@ -80,13 +80,10 @@ do
     esac
 done
 
-APP_HOME=$( cd "${APP_HOME:-./}" && pwd -P ) || exit
-
-APP_NAME="Gradle"
```

Comment:
does it not need to specify appname anymore?

Reply:
This is autogenerated.  I just roll with whatever they suggest.